### ISSUE TITLE
Test SCRUM / Question numbers in Progress bar is 

### ISSUE THREAD

**Issue by GNatala** [scholokov/qax-portal-2#602]
**Steps to reproduce:**

1. Open https://www.qax-camp.com.ua/quizzes/scrum/
2. Click on "Start Тест" under the picture.
3. Test is opened.
4. Pay attention to the font of the question numbers in Progress bar under the timer.

**Actual result:**
Font of the question **numbers in Progress bar is Raleway**.

**Expected result:**
Font of the question **numbers in Progress bar is Rubik**.

**Environment:**
Windows 7 Home edition (x64),
Google Chrome v 108.0.5359.125 (x64)

**Additional іnformation:** https://www.figma.com/file/UyTNSS6KCOdphdTev1HrLF/Portal-design?node-id=931%3A91&t=Ea1k4niGY92Vjp1p-0

**Attachment:**

**Comment by GNatala** [comment]
#96

**Comment by DmytroMarkulych** [comment]
@GNatala Давайте перефоразуємо Summary аким чином, щоб не використовувати there are

**Comment by GNatala** [comment]
@DmytroMarkulych Зазделегідь знала, що так не пишуть, але навіть не могла вигадати, що написати, якщо цифри були - і зникли. Змінила на "no".

**Comment by scholokov** [comment]
please retest

**Comment by GNatala** [comment]
An issue is reproduced.

**Actual result:**
**No question numbers** in Progress bar.

**Expected result:**
**There are question numbers** in Progress bar.

**Environment:**
**Only desktop:**  Windows 7 Home edition (x64),
Google Chrome v 108.0.5359.125 (x64), Firefox 111.0.1

**Attachment:**
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.03.26-14_25_34.webm](https://user-images.githubusercontent.com/101987790/227773281-79912168-88e3-4f08-8608-a6d2b9f65980.webm)

![image](https://user-images.githubusercontent.com/101987790/227773241-502325e5-4657-492f-a264-0bb31ed63a70.png)

**Comment by scholokov** [comment]
please retest with clear cache

**Comment by GNatala** [comment]
An issue is reproduced. **Сache was cleaned.**

**Actual result:**
**No question numbers** in Progress bar.

**Expected result:**
**There are question numbers** in Progress bar.

**Environment:**
 Windows 7 Home edition (x64),
Google Chrome v 108.0.5359.125 (x64), Mozilla 111.0.1, мобільних Android 8.1.0 Chrome 111.0.5563.116

**Attachment:**
![image](https://user-images.githubusercontent.com/101987790/229054207-6b32077c-2d4f-44e3-aedb-0696edfec443.png)

**Comment by scholokov** [comment]
пробували чистити кеш?

**Comment by GNatala** [comment]
Так, в мобільній версіі нумерація вже є. В Windows 7 Home edition (x64), Google Chrome v 108.0.5359.125 (x64), Mozilla 111.0.1 немає

**Comment by GNatala** [comment]
Дякую, вже і таке пробувала:
**Attachment:**
![image](https://user-images.githubusercontent.com/101987790/229881248-2cdd85ee-9c7f-4eac-8316-697a30744007.png)
